mpa/modules/hooks/auxiliary_hooks.py

```diff
@@ -132,6 +134,111 @@ def __exit__(self, exc_type, exc_value, traceback):
         self._handle.remove()
 
 
+class SaliencyMapHookDet(SaliencyMapHook):
```

Comment:
`DetSaliencyMapHook` is more attractable to me. Because this is also a kind of `Hook`.

Reply:
I agree. Renamed